### DISCUSSION TITLE
are those discussion like forums?

### DISCUSSION THREAD

**Discussion by os3albert** [Schweinepriester/github-profile-achievements#289] [Q&A] [answered]
hi guys I'm new, and I'm trying to learn more about github and how it works

**Comment by melvincwng** [comment, marked as answer]
@os3albert Hi Albert, if you need help regarding GitHub related stuff, considering asking the community in the Community Forums over [here](https://github.com/community/community/discussions/28827) instead.

There is a much wider group of people in the Official GitHub Community Forums to address your github related concerns.

Hope this clarifies.

If this answer is helpful, please kindly 'Mark as answer' so that others are aware this issue is addressed.

**Reply by os3albert** [reply]
Ok man thank you very much
